docker/prodlike/config/signer-config.toml

```diff
@@ -0,0 +1,261 @@
+# Default: <none>
+# Required: true
+# Environment: SIGNER_BLOCK_NOTIFIER__SERVER
+server = "tcp://localhost:60401"
+retry_interval = 10
+max_retry_attempts = 5
+ping_interval = 60
+subscribe_interval = 10
+
+# !! ==============================================================================
+# !! Stacks Node Configuration
+# !! ==============================================================================
+[stacks]
+# The RPC URL(s) of the Stacks node(s) to connect to. At least one must be
+# provided. If multiple nodes are provided they will be tried in order when
+# making requests.
+endpoints = ["stacks-node-rpc"]
+
+# This is the start height of the first EPOCH 3.0 block on the stacks
+# blockchain.
+nakamoto_start_height = 867867
+
+# !! ==============================================================================
+# !! Signer Configuration
+# !! ==============================================================================
+[signer]
+# The private key associated with the signer. This is used to generate the
+# signers associated public key and sign messages to other signers.
+#
+# This may be either in 32- or 33-byte format. If you generated the key using
+# `stacks-cli` or other ecosystem tools, it is likely that the key is in 33-byte
+# format which includes a stacks-proprietary suffix byte. The sBTC signer doesn't
+# make use of this byte and it will be trimmed automatically if provided.
+#
+# Format: "<hex-encoded-private-key>" (64 or 66 hex-characters)
+# Required: true
+# Environment: SIGNER_SIGNER__PRIVATE_KEY
+private_key = "your-private-key"
+
+# Specifies which network to use when constructing and sending transactions
+# on stacks and bitcoin. This cooresponds to the `chain` flag in the
+# bitcoin.conf file of the connected bitcoin-core node, and the
+# `burnchain.mode` flag int he config.toml of the connected stacks-core
+# node.
+#
+# Required: true
+# Possible values: mainnet, testnet, regtest
+# Environment: SIGNER_SIGNER__NETWORK
+network = "mainnet"
+
+# The address that deployed the sbtc smart contracts.
+#
+# Required: true
+# TODO(715): Change after SCs have been deployed.
+deployer = "SNYourDeployerAddress"
+
+# The signer database endpoint (pgsql connection string)
+#
+# Required: true
+# Environment: SIGNER_SIGNER__DB_ENDPOINT
+#
+# Defined in the provided docker compose, do not uncomment unless you know
+# what you're doing.
+# -------------------------------------------------------------------------
+# db_endpoint = "postgresql://postgres:postgres@localhost:5432/signer"
+
+# A complete list of (compressed) public keys for known bootstrap signer
+# peers who are approved to be in the sBTC signer set.
+#
+# Required: true Environment: SIGNER_SIGNER__BOOTSTRAP_SIGNING_SET
+# TODO(715): Change after initial signing set has been determined.
+bootstrap_signing_set = [
+    "03providedsigningsetpublickey023a1d53bc96ad670bfe03adf8a06c52e6380",
+    "02providedsigningsetpublickey023b28143130a18099ecf094d36fef0f6135c",
```

Comment:
I'd leave this empty

Reply:
Ah yes, agreed. It made more sense for docs imo

Reply:
Done.